tracking/tracking_tests/traffic_info_test.cpp

```diff
@@ -0,0 +1,54 @@
+#include "testing/testing.hpp"
+
+#include "tracking/traffic_info.hpp"
+
```

Comment:
done for algorithm
vector is included in traffic_info.hpp and I do not use it explicitly here